lib/collector/remote-method.js

```diff
@@ -88,7 +90,32 @@ RemoteMethod.prototype.invoke = function invoke(payload, nrHeaders, callback) {
       if (err) {
         return callback(err)
       }
-      this._post(serialized, nrHeaders, callback)
+      this._post(
+        serialized,
+        nrHeaders,
+        function onResponse(err, res) {
+          if (err) {
+            return callback(err)
+          }
+          if (this._metrics) {
+            const payloadByteLength = byteLength(serialized)
+            const responseByteLength = res.payload ? byteLength(JSON.stringify(res.payload)) : 0
+            this._metrics.measureBytes(
+              `${NAMES.SUPPORTABILITY.NODEJS}/Collector/Output/Bytes`,
+              payloadByteLength,
+              responseByteLength,
+              true
+            )
+            this._metrics.measureBytes(
+              `${NAMES.SUPPORTABILITY.NODEJS}/Collector/${this.name}/Output/Bytes`,
+              payloadByteLength,
+              responseByteLength,
+              true
+            )
+          }
+          return callback(null, res)
```

Comment:
i think here should just always return `callback(err, res)` based on my comment above

Reply:
Fair enough!